app/components/Header/index.tsx

```diff
@@ -83,10 +83,12 @@ const AccountDropdownItems = ({ onClose }: AccountDropdownItemsProps) => {
   );
 };
 
+
 const UpcomingMeetingButton = () => {
+  
   const upcomingMeetingId = useAppSelector(selectUpcomingMeetingId);
   const navigate = useNavigate();
-
+  
```

Comment:
`prettier` will complain about this. Run `yarn prettier` to auto-fix

Reply:
Yep ran the command should be good now:)